pkg/config/validation/validation.go

```diff
@@ -3623,3 +3624,28 @@ func validateWasmPluginSHA(plugin *extensions.WasmPlugin) error {
 	}
 	return nil
 }
+
+func validateWasmPluginVmConfig(vm *extensions.VmConfig) error {
+	if vm == nil {
+		return nil
+	}
+
+	keys := sets.NewSet()
+	for _, env := range vm.Env {
+		if env.Name == "" {
+			return fmt.Errorf("invalid env name")
```

Comment:
What the error message going to look like? Want to make sure the output has enough information that could point user to the `env` at Wasm config.

Reply:
it will looks like
```console
Error from server: error when creating "wasmplugin.yaml": admission webhook "validation.istio.io" denied the request: configuration is invalid: invalid env name
```

Reply:
add the name back

Reply:
> we don't do that for most resources. If I understood right that you mean to give the name/namespace of the WasmPlugin

Yeah. Any reason why we don't do that? What if there are multiple wasm plugins being updated, isn't it hard to tell which one is having problem?